package.json

```
@@ -111,17 +111,17 @@
     "@bbc/psammead-topic-tags": "1.0.0",
     "@bbc/psammead-useful-links": "3.0.20",
     "@bbc/psammead-visually-hidden-text": "2.0.7",
-    "@bbc/web-vitals": "1.0.8",
+    "@bbc/web-vitals": "1.0.9",
```

This is my bad. We can't upgrade this package right now because Opera Mini with extreme data saving mode breaks. I should have created an issue to look into this but been so preoccupied with other things.

```suggestion
    "@bbc/web-vitals": "1.0.8"
```

---

If this had went in I don't think it would have broken Opera Mini because we have the `web-vitals` package pinned in the resolutions field but to be safe we should keep `@bbc/web-vitals` on version 1.0.8

---

Great spot @jroebu14 sorry I missed 😓 

Also a good reason for the seperate PRs, may have been a bit easier to spot.

---

Thanks for the heads up @jroebu14. I have reverted `@bbc/web-vitals` back to `1.0.8`